finetrainers/dataset.py

```diff
@@ -139,7 +141,10 @@ def __getitem__(self, index: int) -> Dict[str, Any]:
         if video_path.suffix.lower() in [".png", ".jpg", ".jpeg"]:
             video = self._preprocess_image(video_path)
         else:
-            video = self._preprocess_video(video_path)
+            try:
+                video = self._preprocess_video(video_path)
+            except:
+                raise ValueError(f"Exception while preprocessing {video_path}.")
```

Comment:
This was added for personal reasons, as I had difficulties finding out which file was causing issues